algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -373,6 +373,9 @@ abstract class CMSTest[K: CMSHasher: FromIntLike] extends WordSpec with Matchers
   val EPS = 0.001
   val SEED = 1
 
+  private[this] val maxDepth = 70
```

Comment:
It's a different class and I didn't want to yak shave too much. Just want the minimal change to enable code coverage to work.